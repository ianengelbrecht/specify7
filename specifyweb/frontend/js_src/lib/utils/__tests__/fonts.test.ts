import '../../tests/__mocks__/fonts';

import { getAvailableFonts } from '../fonts';

test('Unavailable fonts are filterd out', () => {
  expect(getAvailableFonts()).toMatchSnapshot();
});
