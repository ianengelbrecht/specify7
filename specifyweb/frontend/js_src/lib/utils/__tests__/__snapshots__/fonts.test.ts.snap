// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Unavailable fonts are filterd out 1`] = `
[
  "Al Nile",
  "American Typewriter",
  "Andale Mono",
  "Apple Braille",
  "Apple Chancery",
  "Apple Color Emoji",
  "Apple Symbols",
  "Arial",
  "Arial Black",
  "Arial Hebrew",
  "Arial Hebrew Scholar",
  "Arial Narrow",
  "Arial Rounded MT Bold",
  "Arial Unicode MS",
  "Avenir Black",
  "Avenir Black Oblique",
  "Avenir Book",
  "Avenir Heavy",
  "Avenir Light",
  "Avenir Medium",
  "Avenir Next Condensed Demi Bold",
  "Avenir Next Condensed Heavy",
  "Avenir Next Condensed Medium",
  "Avenir Next Condensed Ultra Light",
  "Avenir Next Demi Bold",
  "Avenir Next Heavy",
  "Avenir Next Medium",
  "Avenir Next Ultra Light",
  "Ayuthaya",
  "Bahnschrift",
  "Bangla MN",
  "Bangla Sangam MN",
  "Baskerville",
  "Bodoni Ornaments",
  "Calibri",
  "Calibri Light",
  "Cambria",
  "Cambria Math",
  "Candara",
  "Candara Light",
  "Chalkboard",
  "Chalkduster",
  "Charter Black",
  "Cochin",
  "Comic Sans MS",
  "Consolas",
  "Constantia",
  "Copperplate",
  "Corbel",
  "Corbel Light",
  "Corsiva Hebrew",
  "Courier",
  "Courier New",
  "Devanagari MT",
  "Devanagari Sangam MN",
  "Didot",
  "Ebrima",
  "Euphemia UCAS",
  "Franklin Gothic Medium",
  "Gabriola",
  "Gadugi",
  "Galvji",
  "GB18030 Bitmap",
  "Geneva",
  "Georgia",
  "Gill Sans",
  "Gujarati MT",
  "Gujarati Sangam MN",
  "Gurmukhi MN",
  "Gurmukhi MT",
  "Gurmukhi Sangam MN",
  "Helvetica",
  "Helvetica Neue",
  "Herculanum",
  "Hiragino Maru Gothic ProN W4",
  "Hiragino Mincho ProN W3",
  "Hiragino Mincho ProN W6",
  "Hiragino Sans GB W3",
  "Hiragino Sans GB W6",
  "Hiragino Sans W0",
  "Hiragino Sans W1",
  "Hiragino Sans W2",
  "Hiragino Sans W3",
  "Hiragino Sans W4",
  "Hiragino Sans W5",
  "Hiragino Sans W6",
  "Hiragino Sans W7",
  "Hiragino Sans W8",
  "Hiragino Sans W9",
  "Hoefler Text",
  "Hoefler Text Ornaments",
  "HoloLens MDL2 Assets",
  "Impact",
  "InaiMathi",
  "Ink Free",
  "Javanese Text",
  "Kannada MN",
  "Kannada Sangam MN",
  "Khmer MN",
  "Khmer Sangam MN",
  "Kohinoor Bangla",
  "Kohinoor Telugu",
  "Krungthep",
  "Lao MN",
  "Lao Sangam MN",
  "Leelawadee UI",
  "Lucida Console",
  "Lucida Grande",
  "Lucida Sans Unicode",
  "Luminari",
  "Malayalam MN",
  "Malayalam Sangam MN",
  "Malgun Gothic",
  "Marlett",
  "Microsoft Himalaya",
  "Microsoft JhengHei",
  "Microsoft JhengHei Light",
  "Microsoft JhengHei UI",
  "Microsoft JhengHei UI Light",
  "Microsoft New Tai Lue",
  "Microsoft PhagsPa",
  "Microsoft Sans Serif",
  "Microsoft Tai Le",
  "Microsoft YaHei",
  "Microsoft YaHei Light",
  "Microsoft YaHei UI",
  "Microsoft YaHei UI Light",
  "Microsoft Yi Baiti",
  "MingLiU_HKSCS-ExtB",
  "MingLiU-ExtB",
  "Monaco",
  "Mongolian Baiti",
  "MS Gothic",
  "MS PGothic",
  "MS UI Gothic",
  "Mshtakan",
  "MuktaMahee Bold",
  "MuktaMahee Light",
  "MuktaMahee Medium",
  "MuktaMahee Regular",
  "MuktaMahee SemiBold",
  "MV Boli",
  "Myanmar MN",
  "Myanmar Sangam MN",
  "Myanmar Text",
  "New Peninim MT",
  "Nirmala UI",
  "Noto Nastaliq Urdu",
  "Noto Sans Kannada Black",
  "Noto Sans Kannada ExtraBold",
  "Noto Sans Kannada ExtraLight",
  "Noto Sans Kannada Light",
  "Noto Sans Kannada Medium",
  "Noto Sans Kannada SemiBold",
  "Noto Sans Kannada Thin",
  "Noto Sans Myanmar Light",
  "Noto Sans Myanmar Thin",
  "Noto Sans Oriya",
  "NSimSun",
  "Oriya MN",
  "Oriya Sangam MN",
  "Palatino",
  "Palatino Linotype",
  "Papyrus",
  "Plantagenet Cherokee",
  "PMingLiU-ExtB",
  "PT Mono",
  "PT Sans",
  "PT Sans Caption",
  "PT Sans Narrow",
  "PT Serif",
  "PT Serif Caption",
  "Raanana",
  "Rockwell",
  "Sathu",
  "Segoe MDL2 Assets",
  "Segoe Print",
  "Segoe Script",
  "Segoe UI",
  "Segoe UI Black",
  "Segoe UI Emoji",
  "Segoe UI Historic",
  "Segoe UI Light",
  "Segoe UI Semibold",
  "Segoe UI Symbol",
  "Shree Devanagari 714",
  "SignPainter-HouseScript",
  "Silom",
  "SimSun",
  "SimSun-ExtB",
  "Sinhala MN",
  "Sinhala Sangam MN",
  "Sitka Banner",
  "Sitka Display",
  "Sitka Heading",
  "Sitka Small",
  "Sitka Subheading",
  "Sitka Text",
  "Snell Roundhand",
  "STSong",
  "Sylfaen",
  "Symbol",
  "Tahoma",
  "Tamil MN",
  "Tamil Sangam MN",
  "Telugu MN",
  "Telugu Sangam MN",
  "Thonburi",
  "Times New Roman",
  "Trattatello",
  "Trebuchet MS",
  "Verdana",
  "Webdings",
  "Wingdings",
  "Wingdings 2",
  "Wingdings 3",
  "Yu Gothic Light",
  "Yu Gothic Medium",
  "Yu Gothic Regular",
  "Yu Gothic UI Light",
  "Yu Gothic UI Regular",
  "Yu Gothic UI Semibold",
  "Zapf Dingbats",
  "Zapfino",
]
`;
